layer1/layer1.yaml

```diff
@@ -65,9 +67,21 @@ components:
         flow_control:
           $ref: '#/components/schemas/Layer1.FlowControl'
     
+    Layer1.EthernetVirtual:
+      description: >-
+        Container for virtual ethernet settings
+      type: object
+      properties:
+        speed:
+          description: >-
+            This is the speed that will be set on the virtual ethernet interface.
+          type: string
+          enum: [one_hundred_mbps, one_gbps, ten_gbps, twenty_five_gpbs, fity_gbps, one_hundred_gpbs]
+          default: one_gbps
```

Comment:
We might want the default to be ten_gbps.  @ashutshkumr, thoughts?

Reply:
Either one of `one_gbps` and `ten_gbps` is fine IMO.